src/third-parties/google-analytics/data.json

```diff
@@ -11,7 +11,7 @@
       "action": "append"
     },
     {
-      "code": "window.dataLayer=window.dataLayer||[];window.gtag=function gtag(){window.dataLayer.push(arguments);};gtag('js',new Date());gtag('config','${args.id}')",
+      "code": "window.dataLayer=window.dataLayer||[];window.gtag=function gtag(){window.dataLayer.push(...arguments);};gtag({'gtm.start': Date.now(), 'event': 'gtm.js'});gtag({'config':'{{id}}'})",
```

Comment:
this part was a little weird, I don't know what the "config" does, but I kept it in there. 

screenshot of ga tags applied to html (without t
<img width="606" alt="Screenshot 2023-10-09 at 12 41 58 PM" src="https://github.com/GoogleChromeLabs/third-party-capital/assets/372973/39e14922-4c6c-4ff6-a43c-fb926c323644">
his implementation):

screenshot of the modified code block (looks pretty close):
<img width="567" alt="Screenshot 2023-10-09 at 12 42 09 PM" src="https://github.com/GoogleChromeLabs/third-party-capital/assets/372973/23b91392-e2c8-4ab6-a2c6-c3d1b458c607">

Reply:
updated screenshots - reverted to gtag usage:

snippet:
<img width="1141" alt="Screenshot 2023-10-09 at 5 36 31 PM" src="https://github.com/GoogleChromeLabs/third-party-capital/assets/372973/927fba27-92e9-4ccb-8259-6f2b14d5f402">

generated:
<img width="1134" alt="Screenshot 2023-10-09 at 5 36 46 PM" src="https://github.com/GoogleChromeLabs/third-party-capital/assets/372973/068821f6-f29f-4cfd-9888-984852533db8">

Reply:
weird thing is that if we call the gtag function (either snippet or generated), it's basically an array, vs the automatic events fired are objects 🤷 . 

I did compare with https://developers.google.com/tag-platform/gtagjs/configure and it looks right.